io_scene_vox.py

```diff
@@ -175,6 +194,7 @@ def import_vox(path, *, voxel_spacing=1, voxel_size=1,
                 binary = bin(prop_bits)
                 # Need to read property values, but this gets fiddly
                 # TODO: finish implementation
+                vox.seek(_offset + 12 + s_self, 0)
```

Comment:
Can we discuss this a little more closely.

I don't see where the value of _offset would be coming from.

Looking at the old .vox spec, I do see that there is a "normalized property value" at the end of this chunk. I don't, however see where the N in "4 x N". I presume that's what you're combating here, and using the known size of self to handle it?

Elsewhere I have used a `vox.read(12)` to pass over 12 bytes, rather than using seek, would be good to have consistency one way or the other. seek's second argument, which you've set to 0 here could be 1, which would seek relative to current position.

Perhaps this line could read
```suggestion
                # We have read 16 bytes of this chunk so far, ignoring remainder
                vox.read(s_self - 16)
```

Reply:
I see you made changes to address this in [https://github.com/RichysHub/MagicaVoxel-VOX-importer/pull/8/commits/662f55ab9a3459ccb74d99b3efe4b3c489db4711](this commit), but I'm not totally happy with that solution.

Recreating an `_offset` every chunk, when we only use it in the MATT seems fruitless to me, especially when we can just read ahead the remaining bytes in s_self.

Reply:
Ouch, you are right: here I decided to skip the MATT chunk at all, as you implentation was still "in progress", but I forgot to copy the ftell function call before... 

In pseudo code:

```
while(true)
{
    _offset=ftell()
    ...
    switch(chunk type)
    {
      case PACK:
        ...parse it...
        break
      case MATT:
        fseek(_offset + 12 + chunk size, 0)
        break
      case ...:
      ...
    }
}
```

I tested different files, for example [red_booth_transparency.vox](https://github.com/kluchek/vox-models/blob/master/%23redbooth/%23red_booth_transparency.vox)



```
Reading at dec: 20 hex: 14
Parsing chunk 'PACK':
Reading at dec: 36 hex: 24
Parsing chunk 'SIZE':
Reading at dec: 60 hex: 3C
Parsing chunk 'XYZI':
Reading at dec: 12748 hex: 31CC
Parsing chunk 'RGBA':
Reading at dec: 13784 hex: 35D8
Parsing chunk 'MATT':
Reading at dec: 13832 hex: 3608
Parsing chunk 'MATT':
Reading at dec: 13872 hex: 3630
Parsing chunk 'MATT':
Reading at dec: 13916 hex: 365C
Parsing chunk 'MATT':
Reading at dec: 13964 hex: 368C
Loaded voxels: 3167
Linked voxels: 3000
```